geotrellis-uberjar/project/build.scala

```diff
@@ -0,0 +1,10 @@
+import sbt._
+import scala.util.Properties
+
+object Version {
+  def either(environmentVariable: String, default: String): String = Properties.envOrElse(environmentVariable, default)
+
+  lazy val hadoop  = either("SPARK_HADOOP_VERSION", "2.6.2")
+  lazy val spark   = either("SPARK_VERSION", "1.6.1")
```

Comment:
Forgive me, but is this right?  Shouldn't all references to spark version be 2.0.0?